tests/test_packages/functional/package_test/test_regression_diff.py

```diff
@@ -89,18 +90,34 @@ def test_get_diff_res_count_unresolved(self):
         base_run_id = runs[0].runId
         new_run_id = runs[1].runId
 
+        base_count = self._cc_client.getRunResultCount(base_run_id, [])
```

Comment:
This functional test tests only the diff api. The result count api has different tests cases.
If this test asserts (getDiffResultCount) it can be useful to see the result count for the two separate runs which can help debugging what caused the test to fail.